### PR TITLE
Loading screen

### PR DESCRIPTION
**Pull request by fromgonie**
A loading screen that appears after pressing the button 'Get Result'

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -52,34 +56,37 @@
...
 })
 
-const handleRoleChange = (selectedRole: Role) => {
-  try {
-    role.value = selectedRole
-    quizStore.initializeQuizAnswers(selectedRole.knowledges)
-    quizStore.setSelectedRole(selectedRole)
-  } catch (error) {
-    ErrorMessage('Failed to set selected role')
+const handleRoleChange = (RoleName: string) => {
```
Comment (vitorlofonseca) [inline src/views/HomeView.vue:79]:
```suggestion
const handleRoleChange = (roleName: string) => {
```
Variables are camelCase

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -88,11 +95,18 @@ const getResult = async () => {
     ErrorMessage('No data provided')
     return
   }
-  SuccessMessage('Fetching your results...')
+  const loadingInstance = ElLoading.service({
+    lock: true,
+    background: 'rgba(255, 255, 255, 0.5)'
+  })
   try {
     await quizStore.updateQuizResult(quizStore.quizAnswers)
```
Comment (vitorlofonseca) [inline src/views/HomeView.vue:103]: When you have the result of your request, you have to hide the loading screen

File: src/views/HomeView.vue [hunk 3]
```diff
@@ -172,4 +186,8 @@ h2 {
   width: 100px;
   margin-bottom: 20px;
 }
+
+.example-showcase .el-loading-mask {
```
Comment (vitorlofonseca) [inline src/views/HomeView.vue:190]: example-showcase? This el-loading-mask appears to be good in the components.scss

**Review by vitorlofonseca — APPROVED** [review]

**Review by plinionr3 — APPROVED** [review]
Well done

**Pull request merged**